.github/workflows/pullrequest.yml

```
@@ -48,6 +48,12 @@ jobs:
     with:
       main_branch: ${{ github.base_ref }}
 
+  dedupe:
+    uses: ./.github/workflows/dedupe.yml
```

actionlint (1.7.4)

error while parsing reusable workflow "./.github/workflows/dedupe.yml": "workflow_call" event trigger is not found in "on:" at line:4, column:3

(workflow-call)